sympy/solvers/tests/test_solvers.py

```diff
@@ -1775,20 +1776,16 @@ def test_issue_8828():
 
 
 def test_issue_2840_8155():
-    assert solve(sin(3*x) + sin(6*x)) == [
-        0, -pi, pi, 2*pi, -2*I*log(-(-1)**(S(1)/9)), -2*I*log(-(-1)**(S(2)/9)),
-        -2*I*log((-1)**(S(7)/9)), -2*I*log((-1)**(S(8)/9)), -2*I*log(-S(1)/2 -
-        sqrt(3)*I/2), -2*I*log(-S(1)/2 + sqrt(3)*I/2), -2*I*log(S(1)/2 -
-        sqrt(3)*I/2), -2*I*log(S(1)/2 + sqrt(3)*I/2), -2*I*log(-sqrt(3)/2 - I/2),
-        -2*I*log(-sqrt(3)/2 + I/2), -2*I*log(sqrt(3)/2 - I/2),
```

Comment:
@smichr These `log` values evaluate to slightly incorrect results.
For example : `-2*I*log(-(-1)**(S(1)/9))` evaluates to `−5.58505360638185+2.09933628361472e−140*I`.
`-2*I*log(-(-1)**(S(1)/9))` is symbolically correct, but it's evaluation isn't, so I feel it would be better to get solution in `atan` form.